doc/userguide/install.rst

```diff
@@ -90,18 +90,20 @@ Rust support::
 Ubuntu/Debian
 """""""""""""
 
+.. note:: The following instructions require `sudo` to be installed.
```

Comment:
Is the intention here for `sudo` to be italicized? Or for it to be an inline code block?

Reply:
I actually didn't notice I added that.

I should remove it then, yeah?

Reply:
Depends on the intention you wanted to have.  In most cases we use an inline code-block, which is RST is
```
``sudo``
```

Reply:
I actually didn't have any intention.

My brain at the time probably thought it was going to be github markdown 😄.

Reply:
@jasonish can I close the PR and submit another?

Reply:
Sure!